assets/translations/id.json

```diff
@@ -141,223 +141,223 @@
+    "bgTaskStarted": "Tugas latar belakang dimulai - periksa log.",
+    "runBgCheckNow": "Jalankan Pengecekan Pembaruan Latar Belakang Sekarang",
+    "versionExtractWholePage": "Terapkan RegEx ekstraksi string versi ke seluruh halaman",
     "installing": "Memasang",
     "skipUpdateNotifications": "Lewati notifikasi pembaruan",
-    "updatesAvailableNotifChannel": "Pembaruan tersedia",
-    "appsUpdatedNotifChannel": "Aplikasi diperbarui",
-    "appsPossiblyUpdatedNotifChannel": "Pembaruan aplikasi dicoba",
-    "errorCheckingUpdatesNotifChannel": "Kesalahan dalam memeriksa pembaruan",
-    "appsRemovedNotifChannel": "Aplikasi dihapus",
+    "updatesAvailableNotifChannel": "Pembaruan Tersedia",
+    "appsUpdatedNotifChannel": "Aplikasi Diperbarui",
+    "appsPossiblyUpdatedNotifChannel": "Upaya Pembaruan Aplikasi",
+    "errorCheckingUpdatesNotifChannel": "Kesalahan Memeriksa Pembaruan",
+    "appsRemovedNotifChannel": "Aplikasi Dihapus",
     "downloadingXNotifChannel": "Mengunduh {}",
-    "completeAppInstallationNotifChannel": "Selesaikan pemasangan aplikasi",
-    "checkingForUpdatesNotifChannel": "Memeriksa pembaruan",
-    "onlyCheckInstalledOrTrackOnlyApps": "Hanya periksa aplikasi yang dipasang dan ditandai 'Pelacakan Saja' untuk pembaruan",
-    "supportFixedAPKURL": "Mendukung URL APK tetap",
+    "completeAppInstallationNotifChannel": "Selesaikan Pemasangan Aplikasi",
+    "checkingForUpdatesNotifChannel": "Memeriksa Pembaruan",
+    "onlyCheckInstalledOrTrackOnlyApps": "Hanya periksa aplikasi yang terpasang dan aplikasi Hanya Lacak untuk pembaruanan",
+    "supportFixedAPKURL": "Mendukung Tautan APK yang tetap",
     "selectX": "Pilih {}",
-    "parallelDownloads": "Izinkan mengunduh paralel",
+    "parallelDownloads": "Izinkan unduhan paralel",
     "useShizuku": "Gunakan Shizuku atau Sui untuk memasang",
     "shizukuBinderNotFound": "Layanan Shizuku tidak berjalan",
-    "shizukuOld": "Versi Shizuku lama (<11) - harap perbarui",
+    "shizukuOld": "Versi Shizuku lama (<11) - silakan perbarui",
     "shizukuOldAndroidWithADB": "Shizuku berjalan di Android < 8.1 dengan ADB - perbarui Android atau gunakan Sui sebagai gantinya",
     "shizukuPretendToBeGooglePlay": "Atur Google Play sebagai sumber pemasangan (jika Shizuku digunakan)",
     "useSystemFont": "Gunakan font sistem",
-    "useVersionCodeAsOSVersion": "Gunakan versi kode aplikasi sebagai versi yang terdeteksi OS",
+    "useVersionCodeAsOSVersion": "Gunakan versionCode aplikasi sebagai versi yang terdeteksi oleh OS",
     "requestHeader": "Header permintaan",
     "useLatestAssetDateAsReleaseDate": "Gunakan unggahan aset terbaru sebagai tanggal rilis",
-    "defaultPseudoVersioningMethod": "Metode versi semu standar",
-    "partialAPKHash": "Hash APK parsial",
-    "APKLinkHash": "Hash tautan APK",
-    "directAPKLink": "Tautan APK langsung",
-    "pseudoVersionInUse": "Versi semu sedang digunakan",
+    "defaultPseudoVersioningMethod": "Metode Penomoran Versi Semu",
```

Comment:
you forgot the "standard" part. albeit i was thinking about "metode standar dalam penomoran versi semu" is actually better, but it's mouthful. so keep as is i guess.